lib/dpl/provider/pypi.rb

```diff
@@ -64,7 +65,9 @@ def needs_key?
 
       def push_app
         context.shell "python setup.py register -r #{options[:server] || 'pypi'}"
```

Comment:
You might want to remove this line? Unless the project in question has never been registered with PyPI *ever* in it's entire history this line is not important and will negate any security benefits of using twine.